routers/web/web.go

```diff
@@ -470,13 +478,14 @@ func RegisterRoutes(m *web.Route) {
 				})
 			})
 		}, packagesEnabled)
-		m.Get("/organization", user_setting.Organization)
+		m.Get("/organization", userSettingModuleEnabled(user_setting.UserOrganizations), user_setting.Organization)
```

Comment:
So the user can't see `/user/settings/organization`, but he still can create or leave organizations?

<img width="1549" alt="image" src="https://user-images.githubusercontent.com/9418365/206992115-1caa0605-c641-4ab9-94ba-f4708a0e1b87.png">
<img width="1198" alt="image" src="https://user-images.githubusercontent.com/9418365/206992226-10a3f072-1c0f-41ae-9d15-bcba0ebf377f.png">